cachito/workers/config.py

```diff
@@ -93,6 +93,8 @@ class Config(object):
     # other tasks aren't starved when processing a large archive.
     worker_prefetch_multiplier = 1
 
+    cachito_subprocess_timeout = 3600  # 1 hour
```

Comment:
Nitpick: could you move this line to the other `cachito_` options? Ideally, they should be sorted alphabetically